pyproject.toml

```diff
@@ -40,6 +40,11 @@ python = "^3.7.1"
 sphinx-panels = {version = "^0.5.2", optional = true}
 sphinx-rtd-theme = {version = "^0.5.1", optional = true}
 tqdm = "^4.59.0"
+PyYAML = "^5.4.1"
+joblib = "^1.0.1"
+bounding-box = "^0.1.3"
+IProgress = "^0.4"
```

Comment:
what are these dependencies for `IProgress` and `ipywidgets`?

Reply:
Removed the support for notebook for the time being.